sym-aws-go-approvals/approve/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-lambda-go/lambda"
+
+	"github.com/serverless-templates/sym-aws-go-approvals/internal"
```

Comment:
shouldn't this be `/symopsio/serverless-templates/internal`?

Reply:
nah this is the module path as generated in `gomod.sh`